<launch>

   <!-- 2. start ArUco -->
    <node name="aruco_tracker" pkg="aruco_ros" type="charuco">
        <!-- /camera_info 和 /image 做修改 -->
        <remap from="/camera_info" to="/camera/color/camera_info" />
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="true"/>
        <!-- reference_frame 和 amera_frame 做修改 -->
        <param name="reference_frame"    value="camera_color_frame"/>
        <param name="camera_frame"       value="camera_color_frame"/>
        <param name="marker_frame"       value="camera_marker" />
    </node>
</launch>

